tests/docker/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN curl -s "$KAFKA_MIRROR/kafka-streams-2.2.0-test.jar" -o /opt/kafka-2.2.0/lib
 
 # The version of Kibosh to use for testing.
 # If you update this, also update vagrant/base.sy
```

Comment:
nit: shall we update the typo of `vagrant/base.sy`?

Reply:
This is supposed to say "also update vagrant/base.sh".  We did update base.sh later in the PR.

I will correct the typo here.